src/libcore/iter/mod.rs

```diff
@@ -327,8 +327,9 @@ pub use self::sources::{Empty, empty};
 pub use self::sources::{Once, once};
 
 #[stable(feature = "rust1", since = "1.0.0")]
-pub use self::traits::{FromIterator, IntoIterator, DoubleEndedIterator, Extend,
-                       ExactSizeIterator};
+pub use self::traits::{FromIterator, IntoIterator, DoubleEndedIterator, Extend};
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Nah these end up not really meaning anything, especially on reexports. Right now we just validate that they're there and... that's pretty much it. This is probably one of the least egregious violations we have :)